src/gui/ui_settings.c

```diff
@@ -428,6 +433,9 @@ static int move_idx_in_array(char *array[], int count, char *find, int index_dis
   }
 }
 
+#define XSTR(x) STR(x)
+#define STR(x) #x
```

Comment:
do we need this defines??

Reply:
Sorry I forgot to remove those macros. I used them during debugging.